<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BigCalendar</class>
 <widget class="QWidget" name="BigCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="formTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Calendar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">#widget {
	background:#ffffff;
	border-radius:9px;
	border:1px solid #bbbbbb;
}

QWidget {
	background:#ffffff;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true">QTabWidget {
	background:#ffffff;
}

QTabWidget QWidget {
	background:none;
}</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Month View</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>12</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QWidget" name="calendarControls" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true">#calendarControls {
	border-bottom:1px dotted #bbbbbb;
}

QPushButton:pressed {
	background-color: #555555;
	color: #ffffff;
}</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <item>
                    <widget class="QPushButton" name="cal_previous">
                     <property name="styleSheet">
                      <string notr="true">padding:5px;
border:1px solid #cccccc;
border-radius:9px;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="resources.qrc">
                       <normaloff>:/icons/Back 24x24</normaloff>:/icons/Back 24x24</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Month</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="cal_month">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">background:none;</string>
                     </property>
                     <item>
                      <property name="text">
                       <string>January</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>February</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>March</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>April</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>May</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>June</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>July</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>August</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>September</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>October</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>November</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>December</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Year</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="cal_year">
                     <property name="maximum">
                      <number>3000</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="cal_next">
                     <property name="styleSheet">
                      <string notr="true">padding:5px;
border:1px solid #cccccc;
border-radius:9px;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="resources.qrc">
                       <normaloff>:/icons/Forward 24x24</normaloff>:/icons/Forward 24x24</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="calendarData" native="true">
                <property name="styleSheet">
                 <string notr="true">#calendarData {
	border-left:1px solid #dddddd;
	border-top:1px solid #dddddd;
}

#calendarData QWidget {
	background: #fff;
}</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="calendarDataLayout">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Week View</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>12</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QToolButton" name="toolButton">
                     <property name="styleSheet">
                      <string notr="true">padding:5px;
border:1px solid #cccccc;
border-radius:9px;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="resources.qrc">
                       <normaloff>:/icons/Back 24x24</normaloff>:/icons/Back 24x24</iconset>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="monthLabel">
                     <property name="text">
                      <string>January</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QToolButton" name="toolButton_2">
                     <property name="styleSheet">
                      <string notr="true">padding:5px;
border:1px solid #cccccc;
border-radius:9px;</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="resources.qrc">
                       <normaloff>:/icons/Forward 24x24</normaloff>:/icons/Forward 24x24</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="calendarDataWeek" native="true">
                <property name="styleSheet">
                 <string notr="true">#calendarDataWeek QWidget {
	background: #fff;
}

#calendarDataWeek {
	border:1px solid #dddddd;
}</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_3">
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="calendarDataWeekLayout">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
